src/nodes/accessors/TextureNode.js

```diff
@@ -128,7 +128,15 @@ class TextureNode extends UniformNode {
 
 		if ( builder.isFlipY() && ( texture.isRenderTargetTexture === true || texture.isFramebufferTexture === true || texture.isDepthTexture === true ) ) {
 
-			uvNode = uvNode.setY( uvNode.y.oneMinus() );
+			if ( this.sampler ) {
+
+				uvNode = uvNode.setY( uvNode.y.oneMinus() );
```

Comment:
I think we can use `uvNode = uvNode.flipY()` now?

Reply:
done